### PR TITLE
graphite2: patch for CVE-2017-5436

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/graphite2.rb [hunk 1]
```diff
@@ -28,7 +41,7 @@ def install
   test do
     resource("testfont").stage do
       shape = shell_output("#{bin}/gr2fonttest Simple-Graphite-Font.ttf 'abcde'")
-      assert_match /67.*36.*37.*38.*71/m, shape
+      assert_match(/67.*36.*37.*38.*71/m, shape)
```
Comment (ilovezfs) [inline Formula/graphite2.rb]: what's with the parentheses?

Comment (DomT4) [inline Formula/graphite2.rb]: I was about to blame Rubocop but double checking it's actually Ruby getting on my case about it. `ruby -wc` throws:

```
graphite2.rb:31: warning: ambiguous first argument; put parentheses or a space even after `/' operator
Syntax OK
```

Can always revert if you prefer.

Comment (ilovezfs) [inline Formula/graphite2.rb]: Oh boy ... that is an unfortunate warning given how many of these we have without parentheses.

Comment (ilovezfs) [inline Formula/graphite2.rb]: reverted while pulling since thinking about changing all of them was causing me to hyperventilate

**Comment by ilovezfs** [comment]
Thanks Dom

File: Formula/graphite2.rb (hunk 1 above)
Comment (DomT4) [inline Formula/graphite2.rb]: Heh, no worries 😄. I get the feeling we may have suppressed it in Rubocop but not much can be done about `ruby -wc` warnings beside ignoring them.

**Pull request closed without merging**